### PR TITLE
18 collected coins

### LINKED ISSUE #18: Collected amount of Coins
The player can collect the coins and the amount of coins the player has collected should be monitored

### PR DESCRIPTION
**Pull request by mdingjan**
Will close #18 

Ready for review.

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/level/Track.java [hunk 1]
```diff
@@ -128,6 +181,7 @@ public final synchronized void step() {
 		
 		getPlayer().step();
 
+		distance++;
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/level/Track.java]: Misschien dit? Komt meer overeen met de werkelijkheid :)

    distance += UNITS_PER_TICK;

File: src/main/java/nl/tudelft/ti2206/group9/level/State.java [hunk 2]
```diff
@@ -69,6 +60,14 @@ public static int getScore() {
 	public static void setScore(final int newScore) {
 		State.score = newScore;
 	}
+	
+	/**
+	 *
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/level/State.java]: One star too much, if I'm allowed to be perfectionist ;)

**Comment by mdingjan** [comment]
@mpsijm Thank you, I processed your feedback :D

**Comment by MathiasMeuleman** [comment]
So now the score is only based on the coins? I thought only the distance and #coins would be displayed and in the end the score would be determined based on the amount of coins x coin.VALUE and the distance (maybe multiplied with a multiplier).

Furthermore, try to keep the titles of your commitmessages shorter, 50 chars max.

Also try to minimize the checkstyle errors.

**Comment by thervh70** [comment]
@MathiasMeuleman @mdingjan I've linked issue #17 to that problem about the score. It should be fixed ASAP by preferably @mdingjan

**Comment by mdingjan** [comment]
@MathiasMeuleman
- The score is not part of this issue, nor #31.It is part of issue #17. So in order to be clear: I did not change anything to the score. ;) The score implementation was already there. 
We can change the score calculation in a new branch for #17.
- I will take that into account. Maarten told me the same ;)
- I already try to minimize the amount of checkstyle errors. There might be some 'magic numbers' left.. But in my opinion there are currently no more checkstyle errors that I can fix.

**Pull request merged**